### ISSUE TITLE
Updating PC via link does not work

### ISSUE THREAD

**Issue by georgwiese** [powdr-labs/powdr#1204]
See [this example](https://github.com/powdr-labs/powdr/pull/1203/files#diff-4b134ce6893893e2a1bc8a5d7d48f2362b579e611243d6152dad522fb015605bR62):
```
instr jump_to_bootloader_input X = bootloader_inputs.access X, pc' ->;
```

This does not seem to un-constraint the PC in the next row. So Witgen fails in this example.

**Comment by lvella** [comment]
Hopefully closed by #1221.

**Comment by georgwiese** [comment]
Yes, it does, thanks!
